FinSearchUnified/Bundle/SearchBundleFindologic/QueryBuilder/NavigationQueryBuilder.php

```diff
@@ -25,4 +25,10 @@ public function addCategories(array $categories)
         $categoryPath = implode('_', $categories);
         $this->searchNavigationRequest->setSelected('cat', $categoryPath);
     }
+
+    public function addManufactures(array $manufactures)
+    {
+        $manufacturePath = implode('_', $manufactures);
+        $this->searchNavigationRequest->setSelected('vendor', $manufacturePath);
+    }
```

Comment:
- There is no such thing as a manufacturer path, you will need to provide a vendor name
- `selected` should only be one value, so we should only use the first manufacturer. When are multiple IDs provided by Shopware?
- Should be `public function addManufacturer($manufacturerName)`